File: Python/LinkedList.py
# Node class
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

# Linked list class
class LinkedList:
    def __init__(self):
        self.head = None

    # Function to print the linked list
    def printList(self):
        current = self.head
        while(current):
            print(current.data)
            current = current.next        



if __name__ == '__main__':
    list = LinkedList()
    list.head = Node(1)
    second = Node(2)
    third = Node(3)
    fourth = Node(4)
    list.head.next = second
    second.next = third
    third.next = fourth

    list.printList()

File: DS Implementation/LinkedList.cpp
#include<iostream>
using namespace std;

class Node {
  public:
    int data;
    Node* next;
};

void printList(Node* head){
  Node* curr = head;
  while(curr!=NULL){
    cout<<curr->data<<" -> "<<;
    curr = curr->next;
  }
}

int main(){
  Node* head = NULL;
  Node* second = NULL;
  Node* third = NULL;

  head = new Node();
  second = new Node();
  third = new Node();

  head->data = 10;
  second->data = 20;
  third->data = 30;

  head->next = second;
  second->next = third;

  printList(head);

  return 0; 
}